zerver/openapi/zulip.yaml

```diff
@@ -18647,9 +18647,32 @@ paths:
 
                                 **Changes**: New in Zulip 5.0 (feature level 93).
                             can_mention_group:
-                              type: integer
+                              oneOf:
+                                - type: integer
+                                - type: object
+                                  additionalProperties: false
+                                  properties:
+                                    direct_members:
+                                      description: |
+                                        The list of user IDs that are direct members of the user group.
+                                      type: array
+                                      items:
+                                        type: integer
+                                    direct_subgroups:
+                                      description: |
+                                        The list of user group IDs that are direct subgroups of the user group.
+                                      type: array
+                                      items:
+                                        type: integer
```

Comment:
I would be taking another pass at the API documentation.

Reply:
Yeah, I think we want to define it as a type at the bottom of the file, so the markup here can just be an include, plus I guess the `Changes` entry.

Reply:
So, I have kept description separate here for now since the description is not exactly same at all places like we use "permission to mention the new group" in a couple of places like the endpoint to create groups.